ui-admin/src/components/Card.tsx

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import { isEmpty } from 'lodash'
+
+
+/**
+ * Stylized card component for displaying read-only or editable data.
+ */
+export function Card({ children }: { children: React.ReactNode }) {
```

Comment:
Erin has been using the same style card across her designs, so I felt it made sense to consolidate it into a generic card type that we can reuse. It works kind of like the Modal component, with separate Header/Body components. I don't really like the names I chose, though, so I wanted to get thoughts and opinions on how these components should be named - Card feels a bit too generic, but that _is_ what it is, so idk.

Reply:
For context, here's what they look like: 
![image](https://github.com/broadinstitute/juniper/assets/17440010/7fed4f2b-144b-4eb2-88a7-4fbadf7948df)